updated menus for 2020-MWGR#0 + updated scripts to install/remove CMSSW release on hilton

missirol opened the pull request:
* small update to `runFastValidation.sh` script, to use HLT menus and GT deployed in 2020-MWGR#0

* while trying to install a recent CMSSW release on a hilton node, i needed to update the corresponding scripts in order to use `cmspkg` instead of `apt`, along the lines of what is described for example [here](https://github.com/cms-sw/cmspkg) and [here](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SDTCMSSW_aptinstaller#Recipe_for_the_impatient_SH_BASH);

* the scripts were tested to install (and then remove) `CMSSW_11_1_0_pre1` on a hilton machine;

* there is also a small update to ` L1MenuCheck_FromGT.sh`; the previous implementation led to an error when using a local CMSSW installation.

@fwyzard @Sam-Harper in case you want to review

missirol committed:
added bootstrap_cmssw_rpm.sh

missirol committed:
updated bootstrap_cmssw_rpm.sh

missirol committed:
updated remove_cmssw_rpm.sh

missirol committed:
updated srcpath def in L1MenuCheck_FromGT.sh

missirol committed:
updated {bootstrap,remove}_cmssw.sh scripts

missirol committed:
removed bootstrap_cmssw_apt.sh

missirol committed:
added check in L1MenuCheck_FromGT.sh

missirol committed:
minor update to L1MenuCheck_FromGT.sh

fwyzard commented on L1MenuCheck_FromGT.sh:
```diff
@@ -28,17 +28,26 @@ xmlhash=`conddb --db "oracle+frontier://@frontier%3A%2F%2F%28proxyurl%3Dhttp%3A%
 #All this is to get around the fact that conddb dump tries to modify the release base
 #I have tried to make this robust, but I am not very confident. I have no doubt that this will break sometime in the future...
-srcpath=`echo $CMSSW_SEARCH_PATH | sed '0,/:.*$/s/:.*$//'`
+cmsswbase=${CMSSW_BASE}
```
why don't you use `$CMSSW_BASE` directly, instead of assigning it to `cmsswbase` ?

missirol replied:
i think this is because, below, a temporary local area is created and initialized ([here](https://github.com/cms-tsg-fog/hltpro/pull/19/files#diff-217e7cc897d0a82a12a9eec26bc2c34aR42)), thus changing the original value of `$CMSSW_BASE`

fwyzard commented:
I didn't test it, but from a quick look I would say that it looks reasonable.

missirol commented:
thanks for taking a look

missirol committed:
minor update to setup.sh

missirol committed:
updated runFastTrackValidation.sh (menus of 2020-MWGR#0)

missirol committed:
updated and re-tested {bootstrap,remove}_cmssw.sh scripts

missirol commented:
i re-tested the `{bootstrap,remove}_cmssw.sh` scripts on hilton, installing and removing a couple of releases; i noticed that

* `bootstrap` (for installation) was giving an error if executed in a session where `CMSSW_BASE` was already set, so I added a check to stop the script in such a case

* i updated a bit `remove`, because i wasn't sure why we were bootstrapping the area again if it was not already in place (since the purpose of the script is to remove an existing one, afaik); now the script, more simply, checks if the relevant directories exist, and if so, proceeds with the removal (if i missed something, i can of course revert this change)

Sam-Harper commented:
thank you Marino!

missirol commented:
okay, will merge the changes
